pymc/aesaraf.py

```diff
@@ -180,9 +187,13 @@ def change_rv_size(
         size = shape[: len(shape) - rv_node.op.ndim_supp]
         new_size = tuple(new_size) + tuple(size)
 
+    # create the name of the RV's resizing tensor
+    # TODO: add information where the dim is coming from (obseverd, prior, ...)
```

Comment:
Is this todo going to be implemented in this PR or is this for later?

Reply:
yes, I think it would be helpful. Just wanted to make sure I'm on the right track before I spend more time on it